projects/plugins/jetpack/extensions/blocks/ai-assistant/extensions/jetpack-contact-form/components/ai-assistant-bar/index.tsx

```diff
@@ -83,12 +85,15 @@ export default function AiAssistantBar( {
 	const { inputValue, setInputValue, isVisible, assistantAnchor } =
 		useContext( AiAssistantUiContext );
 
+	const focusOnPrompt = () => {
+		// Small delay to avoid focus crash
+		setTimeout( () => {
+			inputRef.current?.focus?.();
+		}, 100 );
+	};
+
 	const { requestSuggestion, requestingState, stopSuggestion, requestingError } = useAiContext( {
-		onDone: () => {
-			setTimeout( () => {
-				inputRef.current?.focus?.();
-			}, 10 );
-		},
+		onDone: focusOnPrompt,
```

Comment:
IMO, this action is too intrusive:

https://github.com/Automattic/jetpack/assets/77539/2d8bd1ce-d642-4bea-af0f-37fad64d5631

In the video above, I request a form. Then, I started to work on the excerpt. The app focuses on the AI Control textarea when the request is done.
Probably we should check if the block is selected before to focus on the textarea element.

Reply:
Makes sense!